.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -26,51 +26,66 @@ body:
       placeholder: |
+        E.g., What happened, and what did you expect to happen? Add images,
+        GIFs, and videos if you have them on hand.
+
         1. Start the Studio app.
         2. Click on 'Add site'.
         3. ...
+
+        Add any information that may be relevant, such as:
+          - Browser/Platform
+          - Theme
+          - Logs/Errors
     validations:
       required: true
-  - type: textarea
-    id: expected
+
+  - type: markdown
     attributes:
-      label: What you expected to happen
-      placeholder: |
-        e.g. The new site form shows up.
-    validations:
-      required: true
-  - type: textarea
-    id: actual
+      value: |
+        <br>
+
+        ## Impact
+        Please help us understand more about the impact of this issue to help determine next steps.
+        If you are unsure about anything, please use your judgment to make an educated guess.
+  - type: dropdown
+    id: impact
     attributes:
-      label: What actually happened
-      placeholder: |
-        e.g. Clicking the 'Add site' button does nothing.
+      label: Site owner impact
+      description: Approximately what percentage of the total users of the platform are impacted? Unsure? Please provide your most educated guess!
+      options:
+        - Less than 20% of all
+        - Between 20% and 60% of all
+        - More than 60% of all
     validations:
       required: true
   - type: dropdown
-    id: users-affected
+    id: severity
     attributes:
-      label: Impact
-      description: Approximately how many users are impacted?
+      label: Severity
+      description: What is the severity of this issue? Please take a look at the descriptions below for further context.<br>
+        <br> - **Critical:** Prevents core functionality or has severe impact on the website/platform.
+        <br> - **Major:** Significantly impairs important features or has notable impact on the website/platform.
+        <br> - **Moderate:** Affects non-critical features or has limited impact on the website/platform.
+        <br> - **Minor:** Causes inconvenience or has minimal impact on functionality.
       options:
-        - One
-        - Some (< 50%)
-        - Most (> 50%)
-        - All
+        - Critical
+        - Major
+        - Moderate
+        - Minor
     validations:
       required: true
   - type: dropdown
-    id: workarounds
+    id: additional-impact
     attributes:
-      label: Available workarounds?
+      label: What other impact(s) does this issue have?
+      description: You may select more than one
       options:
-        - No and the app is unusable
-        - No but the app is still usable
-        - Yes, difficult to implement
-        - Yes, easy to implement
-        - There is no user impact
-    validations:
-      required: true
+        - Platform revenue
+        - Agency or developer revenue
+        - Individual site owner revenue
+        - No revenue impact
```

Comment:
Those are not clear in the case of Studio, which is a desktop app.

Reply:
This section covers one of several "priority change triggers" that help bump the issue's priority based on revenue impact

Are there any Studio-specific exceptions you think should alter the initial priority calculation we could add here vs. any other automation options? More detail on these triggers is in: pfVjQF-su-p2

Reply:
I can't see anything. Should we drop this part?